orca-mine/src/main/groovy/com/netflix/spinnaker/orca/mine/pipeline/CanaryStage.groovy

```diff
@@ -46,18 +48,44 @@ class CanaryStage implements StageDefinitionBuilder, CancellableStage {
   @Override
   def List<Stage> aroundStages(Stage stage) {
     Map canaryStageId = [
-      canaryStageId: stage.id,
-      failPipeline: stage.context.failPipeline,
+      canaryStageId   : stage.id,
+      failPipeline    : stage.context.failPipeline,
```

Comment:
What setting in your editor does this? Would like to adopt.

Reply:
Editor > Code Style > Groovy > List and map literals > Align when multiple